docker/docker-compose.yml

```
@@ -30,6 +30,12 @@ services:
       POSTGRES_DB: tacticalrmm
       POSTGRES_USER: ${POSTGRES_USER}
       POSTGRES_PASSWORD: ${POSTGRES_PASS}
+    healthcheck:
+      test: [ "CMD", "pg_isready", "-U", "tactical", "-d", "tacticalrmm" ]

```

```suggestion
      test: [ "CMD", "pg_isready", "-U", "$$POSTGRES_USER", "-d", "$$POSTGRES_DB" ]

```
Maybe use env's here?